src/pages/certifications/responsive-web-design/basic-css/import-a-google-font/index.md

```diff
@@ -3,8 +3,21 @@ title: Import a Google Font
 ---
 ## Import a Google Font
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/certifications/responsive-web-design/basic-css/import-a-google-font/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+````CSS
```

Comment:
Please use a `html` codeblock for the `link` declaration.

Reply:
I will do so. Thanks